scripts/build.py

```diff
@@ -1119,6 +1160,11 @@ def build():
     safe_delete_directory_tree(THIRD_PARTY_GENERATED_DEV_DIR)
     build_third_party_libs(THIRD_PARTY_GENERATED_DEV_DIR)
 
+    if not options.prod_mode:
```

Comment:
Wouldn't this need to be done in prod mode as well?

Reply:
Yes, that is updated in later commits.